experimental/framework/fluid-static/src/FluidStaticTinylicious.ts

```diff
@@ -0,0 +1,99 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    ITinyliciousFileConfig,
+    TinyliciousService,
+} from "@fluid-experimental/get-container";
+import { Container } from "@fluidframework/container-loader";
+import {
+    DOProviderContainerRuntimeFactory,
+    FluidContainer,
+} from "./containerCode";
+import { ContainerConfig } from "./types";
+
+/**
+ * FluidTinyliciousInstance provides the ability to have a Fluid object backed by a Tinylicious service
+ */
+export class FluidTinyliciousInstance {
```

Comment:
I don't think we need two packages here. It should just be one client package for the tinylicious client bits named how we choose. We can then consolidate the logic.

Reply:
Consolidated logic into `@fluid-experimental/tinylicious-client` package here https://github.com/microsoft/FluidFramework/pull/5838